source/test/aws-instance-scheduler-remote-stack.test.ts

```diff
@@ -13,51 +13,13 @@
  *  permissions and limitations under the License.                            *
  *****************************************************************************/
 
-import { SynthUtils } from '@aws-cdk/assert';
-import * as cdk from 'aws-cdk-lib';
-import instanceSchedulerStack = require('../lib/aws-instance-scheduler-remote-stack');
-import '@aws-cdk/assert/jest';
+ import { SynthUtils } from '@aws-cdk/assert';
```

Comment:
This library is deprecated, consider migrating away: https://github.com/aws-solutions/aws-security-hub-automated-response-and-remediation/pull/87

Reply:
added to backlog